app/src/main/java/emu/skyline/applet/swkbd/SoftwareKeyboardConfig.kt

```diff
@@ -0,0 +1,222 @@
+/*
+ * SPDX-License-Identifier: MPL-2.0
+ * Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+ */
+
+@file:OptIn(ExperimentalUnsignedTypes::class)
+
+package emu.skyline.applet.swkbd
+
+import emu.skyline.utils.*
+
+val DownloadCodeCodepoints = intArrayOf('0'.code, '1'.code, '2'.code, '3'.code, '4'.code, '5'.code, '6'.code, '7'.code, '8'.code, '9'.code, 'A'.code, 'B'.code, 'C'.code, 'D'.code, 'E'.code, 'F'.code, 'G'.code, 'H'.code, 'J'.code, 'K'.code, 'L'.code, 'M'.code, 'N'.code, 'P'.code, 'Q'.code, 'R'.code, 'S'.code, 'T'.code, 'U'.code, 'V'.code, 'W'.code, 'X'.code)
+val OutsideOfMiiNicknameCodepoints = intArrayOf('@'.code, '%'.code, '\\'.code, 'ō'.code, 'Ō'.code, '₽'.code, '₩'.code, '♥'.code, '♡'.code)
+
+//this matches KeyboardMode in skyline/applet/swkbd/software_keyboard_config.h
```

Comment:
```suggestion
/**
 * This data class matches KeyboardMode in skyline/applet/swkbd/software_keyboard_config.h
 */
```
Basically, convert to javadoc (or well, kotlindoc).
Do the same for all instances in this file.